CloneTest/Program.cs

```diff
@@ -1,5 +1,214 @@
 ﻿// See https://aka.ms/new-console-template for more information
 
+/*
 string? text = null;
 text ??= "Jeśli widzisz ten tekst to znaczy, że wszystko działa poprawnie";
-Console.WriteLine(text);
\ No newline at end of file
+Console.WriteLine(text);
+*/
+
+
+/*
+int? rozmiar_gestosc_x = null;
+
+double[,] gestosci = { { } };
```

Comment:
Jak nie potrzebujesz nic objaśniać/tłumaczyć w kodzie to usuwaj zakomentowany kod

Reply:
Dzięki za komentarz. Posprzątałem